app/models/config.rb

```diff
@@ -65,4 +66,10 @@ def self.start_day
     start ||= "monday"
     start.downcase.to_sym
   end
+
+  def self.show_practical_support?
+    show_value = Config.find_or_create_by(config_key: 'show_practical_support').options.try :last
+    show_value ||= "Yes"
```

Comment:
Since we're only checking for equivalence to no, I don't think it's necessary to use an or-equals operator here to set the value. I would suggest optimizing for readability and stripping this line in favor of `|| ''` on the line above, or packing this into one expression instead of three (e.g. `Config.find_or_create_by(...).options.try(...).to_s.downcase.strip.to_sym != :no`).

Reply:
I really only included this because nil.downcase is undefined so I'd have to do nil.to_s.downcase, and then turn the nil string into a symbol, which I'm not confident there aren't dragons lurking if I do it. I considered digging or using a ton of safe navs, but it felt sloppy.

I also liked keeping with the existing pattern from my previous PR when possible to reduce cognitive overhead.